components/lists/Signaling.tsx

```diff
@@ -19,76 +20,98 @@ interface SignalingProps {
 }
 
 const Signaling = forwardRef<SignalingRef, SignalingProps>((props, ref) => {
-  const displayUsers = props.users?.length ? props.users : defaultUsers
+  const availableUsers = props.users?.length ? props.users : onlineUsers
+  const otherusers = props.users?.length ? props.users : offlineUsers
```

Comment:
- does this mean that both availableUsers and otherUser can have the same list of users? 
- can we rename otherusers to `offlineUsers`, that's what they are supposed to be right?
- and also instead of checking that we have user from props everytime we need them, can we redestructure them in the props and give it an default value of emty array to make user we always have them, something like this,

```suggestion
const Signaling = forwardRef<SignalingRef, SignalingProps>(({ users=[] }, ref) => {
  const availableUsers =users?.length ? users : onlineUsers
  const otherusers = users?.length ?users : offlineUsers
```
- I don't get why we can't just have onlineUsers and offlineUsers, instead of renaming them here